infra.lvm_tools/shrink_lv/roles/shrink_lv/tasks/check_device.yaml

```diff
@@ -0,0 +1,20 @@
+- name: Get the mount point info
+  ansible.builtin.set_fact:
+    _mount_info: "{{ ansible_facts.mounts | selectattr('device', 'equalto', item.device) }}"
```

Comment:
Does it matter if the device has a mount point or that it's mounted when the facts are retrieved? I think this check should only care that the device name exists in the host. WDYT?

Reply:
Also, an LV can be found as `/dev/mapper/<vg_name>-<lv_name>` as well as `/dev/<vg_name>/<lv_name>`. This check would render the later one as not valid.

Reply:
You're right about the fact that its enough that is exists. But, we use the `mount_info` for additional verifications such as the filesystem type and bytes used. So, if we don't get the path that is actually mounted, we can't check that.